xil_carma_basic_lanefollow_scenario/cdasim_config/carma/README.md

```diff
@@ -0,0 +1 @@
+This .osm map file needs to be copied over to /opt/carma/maps
```

Comment:
This information is already captured in the Readme for the config. Seems like this readme is not adding any additional value at the moment. We can either remove it or update it to include some high level information about the configuration parameters available here.

Reply:
ah okay, removing it lol